sklearn/metrics/tests/test_classification.py

```diff
@@ -1374,7 +1374,27 @@ def test_log_loss():
     loss = log_loss(y_true, y_pred)
     assert_almost_equal(loss, 1.0383217, decimal=6)
 
+    #test labels option
+
+    X = [[1,1], [1,1], [2,2], [2,2]]
+    y_label = [1,1,2,2]
+
+    X_test = [[2,2], [2,2]]
+    y_true = [2,2]
+    y_score = np.array([[0.1,0.9], [0.1, 0.9]])
+    
+    # because y_true label are the same, if not use labels option, will get error
+    error_logloss = log_loss(y_true, y_score)
+    label_not_of_2_loss = -np.mean(np.log(y_score[:,0]))
+    assert_almost_equal(error_logloss, label_not_of_2_loss)
+    
+
+    # use labels, it works
+    label_of_2_loss = -np.mean(np.log(y_score[:, 1]))
+    expected_logloss = log_loss(y_true, y_score, labels=[1, 2])
```

Comment:
I would rename `label_of_2_loss` to `true_log_loss` and `expected_logloss` to maybe `calculated_log_loss`